src/Orchard.Cms.Web/Modules/Orchard.OpenId/AdminMenu.cs

```diff
@@ -21,20 +21,18 @@ public void BuildNavigation(string name, NavigationBuilder builder)
             }
 
             builder
-                .Add(T["Content"], content => content
-                    .Add(T["OpenID Connect Apps"], "5", installed => installed
-                        .Action("Index", "Admin", "Orchard.OpenId")
-                        .Permission(Permissions.ManageOpenIdApplications)
-                        .LocalNav()
-                    )
-                )
                 .Add(T["Design"], design => design
```

Comment:
I've added this commit while you reviewed just to move OpenId Connect Apps to Design ->Security Section